AK/FixedCharBuffer.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Array.h>
+#include <AK/StringView.h>
+#include <AK/TypedTransfer.h>
+
+namespace AK {
+
+template<size_t Size>
+class FixedCharBuffer {
```

Comment:
This class is called "fixed char buffer" but it is actually just a small wrapper around `Array<u8, Size>`, which is not an array of chars. it's an array of unsigned chars.

Is this supposed to hold chars, or UTF-8?

Reply:
It is supposed to hold ASCII chars (`u8` chars). If you have a better name please suggest it and we can discuss it :)

Reply:
Hm. `FixedStringBuffer` ? `StringBuffer`?

Reply:
`FixedStringBuffer` seems more correct :)